### PR TITLE
Update index.d.ts

### PR DESCRIPTION
**Pull request by guillenotfound**
This accepts a boolean as per the docs, that way it won't load any HTML files.

https://www.npmjs.com/package/send#index

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
Could you please add a test for the type: test/types/index.ts?

File: index.d.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ interface SendOptions {
   etag?: boolean;
   extensions?: string[];
   immutable?: boolean;
-  index?: string[];
+  index?: string[] | boolean;
```
Comment (climba03003) [inline index.d.ts:44]:
```suggestion
  index?: string[] | false;
```

File: index.d.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export interface FastifyStaticOptions extends SendOptions {
...
```
Comment (climba03003) [inline index.d.ts:69]:
```suggestion
  index?: string[] | false;
```

File: index.d.ts (hunk 1 above)
Comment (climba03003) [reply index.d.ts:44]: It should be `false` only. 
https://github.com/pillarjs/send/blob/de073ed3237ade9ff71c61673a34474b30e5d45b/index.js#L1056

File: index.d.ts (hunk 2 above)
Comment (climba03003) [reply index.d.ts:69]: Some as above

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: index.d.ts (hunk 2 above)
Comment (guillenotfound) [reply index.d.ts]: Done 👍

**Comment by guillenotfound** [comment]
> Could you please add a test for the type: test/types/index.ts?

Sure, done ;)

**Review by mcollina — APPROVED** [review]
lgtm

**Review by climba03003 — APPROVED** [review]

**Pull request merged**